rancher2/structure_project.go

```diff
@@ -7,10 +7,10 @@ import (
 
 // Flatteners
 
-func flattenProjectContainerResourceLimit(in *managementClient.ContainerResourceLimit) []interface{} {
-	obj := make(map[string]interface{})
+func flattenProjectContainerResourceLimit(in *managementClient.ContainerResourceLimit) []any {
+	obj := make(map[string]any)
 	if in == nil {
-		return []interface{}{}
+		return []any{}
```

Comment:
This can return `nil` which avoids an allocation? (There are a few of these below).